crates/re_space_view_spatial/src/mesh_cache.rs

```diff
@@ -24,7 +24,12 @@ pub struct MeshCache(ahash::HashMap<MeshCacheKey, Option<Arc<LoadedMesh>>>);
 #[derive(Debug, Clone, Copy)]
 pub enum AnyMesh<'a> {
     Asset(&'a re_types::archetypes::Asset3D),
-    Mesh(&'a re_types::archetypes::Mesh3D),
+    Mesh {
+        mesh: &'a re_types::archetypes::Mesh3D,
+
+        /// If any textures are in the mesh's material, they use this hash for texture manager lookup.
```

Comment:
They don't live in the material anymore though.

```suggestion
        /// If there are any textures associated with that mesh (albedo etc), they use this hash for texture manager lookup.